docs/pages/blog/introducing-mui-base.md

```diff
@@ -0,0 +1,93 @@
+For example, a SwitchUnstyled contains the root, thumb, input, and track slots.
+You can control props passed to each of these slots (including, perhaps most importantly, `className`) based on the component's state or even replace the default slot components with your own.
+
+<img src="/static/blog/introducing-mui-base/switch-slots.png" style="width: 796px; margin-bottom: 24px;" alt="Depiction of SwitchUnstyled components' slots" />
+
+Hooks take this one step further.
+They provide functionality, and developers are free to implement the structure to the DOM elements, their interactions, and look.
+They offer maximum customizability at the cost of requiring more to implement by developers.
+
+Upon calling, a hook returns an object describing the component's state (i.e., whether the switch is turned on) and with methods that apply accessibility props and event handlers.
+You should spread these props on components you defined.
+
+```tsx
+function MySwitch(props: UseSwitchParameters) {
+  const { getInputProps, checked, disabled } = useSwitch(props);
+
+  const stateClasses = {
+    checked,
+    disabled,
+  };
+
+  return (
+    <span className={clsx('root', stateClasses)}>
+      <span className="thumb"} />
+      <input type="hidden" {...getInputProps()} />
+    </span>
+  );
+}
+```
+
+## What's included
+
+We are releasing the initial version of the library with 17 components.
+Their docs are available on [our website](/base/getting-started/overview/).
+
+We are going to work on creating more in the upcoming months.
+You can track our progress (and comment to influence our priorities) on a [dedicated GitHub issue](https://github.com/mui/material-ui/issues/27170).
+
+The @mui/base package is released as an alpha.
+This means the API of the components can still change, especially when we receive feedback about its current state from the community.
+However, we believe the library is solid enough to build design systems with it.
+In fact, we're using MUI Base to create [Joy UI](/blog/first-look-at-joy/).
+In the future, MUI Base will also be used as a foundation of Material UI components so that you can expect consistent quality across our entire suite of products.
+
+## Feedback needed
+
+So far, we've been mostly using MUI Base ourselves.
+Now we need to hear what you think of the library, what you like about it, and what are the areas that could look or work differently.
+Please help us create the best possible headless component library out there!
+
+If you found any bugs, have trouble understanding some concepts, or have ideas for improvements, don't hesitate to open issues on GitHub.
+Make sure to include "[base]" or a component/hook name in the issue title, so we recognize it.
+Also, we're eager to know what you create with MUI Base.
+Please share your creations on Twitter to let us (and others) know what our library can be used for.
```

Comment:
Let's whittle this down to one primary call to action (for feedback), and give installation instructions to encourage readers to install the package right away. We can ask users to share what they're building in social media posts that promote this article.

```suggestion
Give MUI Base a try today by installing the package via npm:

```sh
npm install @mui/base
```

Or yarn:

```sh
yarn add @mui/base
```

And be sure to let us know what you think!
If you find any bugs or want to share ideas for improvements, please don't hesitate to open an issue in the [MUI Core repository on GitHub](https://github.com/mui/material-ui/issues/new/choose).
Be sure to include "[base]" in the issue title to help us keep things organized.
```

Reply:
GH is always weird about including code blocks in review comments. Hopefully you can at least what I was going for there. 😅

Reply:
Yes, thanks!

As for code blocks - you have to use four (or more) backticks to surround the suggestion block if it contains nested code blocks:

`````
````
```
this way you can nest code blocks like there's no tomorrow!
```
````
`````

Reply:
Thank you!! I never knew there was a workaround for that!